atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -165,6 +173,17 @@ def run(self):
                 if image.registry:
                     raise RuntimeError("Image name must not contain registry: %r" % image.registry)
 
+                if not source_oci_image_path:
```

Comment:
What does this condition mean? I'm not familiar with this part of the code

Reply:
source container images are not checked (they are .... different)

Reply:
Ah, so `source_oci_image_path` == `it is a source container build`?

Reply:
yes